job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext/buildResult.groovy

```diff
@@ -0,0 +1,25 @@
+job('example') {
+    triggers {
+        buildResult('H/10 * * * *') {
+            combinedJobs()
+            triggerInfo('My job 1') {
```

Comment:
The first (and undocumented) rule is "keep it simple" :smirk:

The optional vs. mandatory options design rule should avoid long parameter list with optional parameters, e.g. `foo(String one, int two = 0, String three= "bar", boolean four = false)`. I think this case is different, so we can make an exception. Please do the refactoring.